instrumentation/jms/src/test/java/brave/jms/PropertyFilterTest.java

```diff
@@ -15,13 +15,15 @@
 
 import java.util.Collections;
 import javax.jms.JMSException;
+import javax.jms.Message;
 import javax.jms.TextMessage;
 import org.apache.activemq.command.ActiveMQTextMessage;
 import org.junit.After;
 import org.junit.Test;
 
 import static brave.test.util.ClassLoaders.assertRunIsUnloadable;
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
```

Comment:
do you mind undoing wildcard imports? we don't use wildcard anywhere in our projects.

Reply:
Fixed. (shakes fist at IDEs)